docs/Configuration.md

```diff
@@ -253,6 +253,63 @@ EOF
 
 Now that you have configured `provider-gcp` with Workload Identity supported.
 
+## Authenticate with Impersonated Service Account
+#### 0. Setup Variables
+
+```console
+$ CLIENT_GCP_SERVICE_ACCOUNT=<YOUR_CLIENT_GCP_SERVICE_ACCOUNT>                               # e.g.) impersonate-example
+```
+
+#### 1. Setup Provider with Workload Identity
+
+Follow directions [here](#authenticating-with-workload-identity)
```

Comment:
Are we going to follow all the Authentication steps with Workload Identity, could you please make it clear here?